packages/lib-user/src/components/UserStats/components/myStyled.js

```diff
@@ -0,0 +1,6 @@
+import styledComponents from 'styled-components';
```

Comment:
This is madness, maybe fixed with styled-components v6.

Reply:
No wait, this file is converting the CJS version of styled components to ESM, but simply re-exporting the ES module otherwise?